### PR TITLE
Add new container type 96-flat-white-dc

### PR DESCRIPTION
**Pull request by polarpine**

### CODE REVIEW

File: autoprotocol/container_type.py [hunk 1]
```diff
@@ -1197,6 +1197,41 @@ def row_count(self):
     cat_no="353077",
 )
 
+#:
+FLAT96DELTA = ContainerType(
+    name="Nunc MicroWell 96-Well, Nunclon Delta-Treated, Flat-Bottom Microplate",
+    shortname="96-flat-white-dc",
+    is_tube=False,
+    well_coating=None,
```
Comment (joshhacksthings) [inline autoprotocol/container_type.py:1205]: is the Nunclon Delta a well coating treatment?

**Review by joshhacksthings — APPROVED** [review]
looks good, just had one question on well coating

File: autoprotocol/container_type.py (hunk 1 above)
Comment (polarpine) [reply autoprotocol/container_type.py:1205]: It's an "energy-treated surface".  Leaving `None` for now until standardization of "well_coating" properties.

**Comment by samjoch** [comment]
please make sure to add it to the container types service.

**Pull request merged**